web/beacon-app/src/features/members/types/memberFormService.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable prettier/prettier */
+import { useFormik } from 'formik';
+import { object, string } from 'yup';
+
+import { MEMBER_ROLE } from '@/constants/rolesAndStatus';
+
+import { MemberRole, NewMemberDTO } from '../types/memberServices';
+
+export const FORM_INITIAL_VALUES = {
+  email: '',
+  name: '',
+  role: MEMBER_ROLE.ADMIN as MemberRole,
+} satisfies NewMemberDTO;
+
+export const FORM_VALIDATION_SCHEMA = object({
+  email: string().required('Email is required').email('Email is invalid'),
+  role: string().required('role is required'),
```

Comment:
```suggestion
  role: string().required('Role is required'),
```

Reply:
Should `name` also be required?

Reply:
i'm not sure that's why i didnt make it required but let do it now and we will figure out later. thanks